wagtail/admin/views/home.py

```diff
@@ -144,3 +146,10 @@ def default(request):
     redirected to the login page.
     """
     raise Http404
+
+
+@cache_page(31536000)  # 1 year
```

Comment:
To cache in browsers, we should use the [`cache_control`](https://docs.djangoproject.com/en/3.1/topics/http/decorators/#django.views.decorators.cache.cache_control) header. This one is for caching server-side.

Reply:
Yes, it is intended to be server-side. We don't want to render that template each request. 
It loads icons template tag which triggered hooks. 

Matt already experienced the performance hit (currently the same code runs for adding the sprite to the base template). He added this:
https://github.com/wagtail/wagtail/blob/master/wagtail/admin/templatetags/wagtailadmin_tags.py#L551-L562

And that makes this sever-side cache redundant. So I'll drop it.

Reply:
Oh, I see, thanks!